wolfcrypt/src/asn.c

```diff
@@ -2131,20 +2143,31 @@ int GetLength_ex(const byte* input, word32* inOutIdx, int* len, word32 maxIdx,
             minLen = 1 << ((bytes - 1) * 8);
         }
 
-        /* Check the number of bytes required are available. */
-        if ((idx + bytes) > maxIdx) {
-            WOLFSSL_MSG("GetLength - bad long length");
+        /* Check the number of bytes required are available.  At least the
```

Comment:
Change check above b == 0x80 to be b == ASN_INDEF_LENGTH.

Reply:
Certainly!